docs/README.md

```diff
@@ -607,6 +609,54 @@ The easiest way to get started with production-grade metrics collection is to us
 
 Congratulations! You should now be able to see your control plane and data plane metrics in the OpenShift Console. Just go to Observe -> Metrics and try the query `istio_requests_total`.
 
+### Configure tracing with OpenShift distributed tracing
+This section describes how to setup Istio with OpenShift Distributed Tracing to send distributed traces.
+
+*Prerequisites*
+* A Tempo stack is installed and configured
+* An instance of an OpenTelemetry collector is already configured in the istio-system namespace
+* An Istio instance is created with the `openshift` profile
+* An Istio CNI instance is created with the `openshift` profile
+
+*Steps*
+1. Configure Istio to enable tracing and include the OpenTelemetry settings:
+    ```yaml
+    meshConfig:
+      enableTracing: true
+      extensionProviders:
+      - name: otel-tracing
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local 
+    ```
+The *service* field is the OpenTelemetry collector service in the `istio-system` namespace.
+
+2. Create an Istio telemetry resource to active the OpenTelemetry tracer
+    ```yaml
+    apiVersion: telemetry.istio.io/v1
+    kind: Telemetry
+    metadata:
+      name: otel-demo
+      namespace: istio-system
+    spec:
+      tracing:
+      - providers:
+          - name: otel-tracing
+            randomSamplingPercentage: 100
+    ```
+
+3. Validate the integration: Generate some traffic
+
+We can [Deploy Bookinfo](#deploy-gateway-and-bookinfo) and generate some traffic.
+
+4. Validate the integration: See the traces in the UI
+
+```sh
+kubectl get routes -n tempo tempo-sample-query-frontend-tempo.apps-crc.testing
```

Comment:
This this the actual name of the `route` object? `tempo-sample-query-frontend-tempo.apps-crc.testing`

Reply:
This part `.apps-crc.testing` seems like a typo?

Reply:
good catch! Updated now